cmd/server-main.go

```diff
@@ -920,6 +922,16 @@ func serverMain(ctx *cli.Context) {
 		globalNodeNamesHex[hex.EncodeToString(nodeNameSum[:])] = struct{}{}
 	}
 
+	bootstrapTrace("waitForQuorum", func() {
+		result := newObject.Health(context.Background(), HealthOptions{Startup: true})
+		for !result.Healthy {
+			d := time.Duration(r.Float64() * float64(time.Second))
+			logger.Info("Waiting for quorum healthcheck to succeed.. possible cause unhealthy sets (%s), retrying in %s", result, d)
+			time.Sleep(d)
+			result = newObject.Health(context.Background(), HealthOptions{})
```

Comment:
Shalln't add a hidden env variable to bypass this ? probably someone with a very cluster will not mind losing an erasure set and wants to keep his cluster running rather than stuck in the boot mode

Reply:
will add it after.